times_mapping.txt

```diff
@@ -42,7 +42,7 @@ FLO_FUNC[REG,DATAYEAR,PRC,COM_GRP,COM,TSLVL,VALUE] = ~TODO(Region,Year,TechName,
 FLO_FUNCX[REG,DATAYEAR,PRC,COM,COM,VALUE] = ~TODO(Region,Year,TechName,CommName,CommName,TODO-FLO_FUNCX)
 FLO_MARK[REG,DATAYEAR,PRC,COM,BD,VALUE] = ~FI_T(Region,Year,TechName,Comm-OUT,LimType,Attribute:FLO_MARK)
 FLO_SHAR[REG,DATAYEAR,PRC,COM,COM_GRP,ALL_TS,BD,VALUE] = ~FI_T(Region,Year,TechName,Comm-IN,Other_Indexes,TimeSlice,LimType,Attribute:FLO_SHAR)
-FLO_SUB[REG,DATAYEAR,PRC,TSLVL,CUR,VALUE] = ~TFM_INS(Region,Year,TechName,TimeSlice,Curr,Attribute:FLO_SUB)
+FLO_SUB[REG,DATAYEAR,PRC,COM,TSLVL,CUR,VALUE] = ~TFM_INS(Region,Year,TechName,Comm-IN,TimeSlice,Curr,Attribute:FLO_SUB)
```

Comment:
Should it be CommName as the one below?

Reply:
According to merged_tables.txt, ~TFM_INS has the following columns: TimeSlice,LimType,Attribute,Year,PSet_Set,PSet_PN,PSet_PD,PSet_CI,PSet_CO,CSet_Set,CSet_CN,CSet_CD,Region,VALUE,SOW,CommGrp,Stage,AllRegions,Curr,Other_Indexes

I think both are wrong and for TIMS, neither table gets generated because none of TechName, Comm-IN and CommName exist

Reply:
I think these should be Comm-IN